src/Cell.tsx

```diff
@@ -45,24 +57,27 @@ function Cell<R, SR>({
     'rdg-cell',
     {
       'rdg-cell-frozen': column.frozen,
-      'rdg-cell-frozen-last': column.idx === lastFrozenColumnIndex
+      'rdg-cell-frozen-last': column.idx === lastFrozenColumnIndex,
+      'rdg-cell-selected': isSelected,
+      'rdg-cell-copied': isCopied,
+      'rdg-cell-dragged-over': isDraggedOver
```

Comment:
I wonder if we should set all these classes and event handlers at the row level, that way we can pass fewer props down to the Cell. Though that might require some `useCallback` 🤔

Reply:
We probably can. Let's revisit in the next PR